toonz/sources/toonzfarm/tfarmserver/tfarmserver.cpp

```diff
@@ -701,6 +706,11 @@ void FarmServer::queryHwInfo(HwInfo &hwInfo)
 	mib[1] = HW_MEMSIZE;
 	len = sizeof(physMemSize);
 	sysctl(mib, 2, &physMemSize, &len, NULL, 0);
+#endif
```

Comment:
My suggestion here was to have
```
#if defined(WIN32)
…
#elif defined(__sgi)
…
#elif defined(MACOSX)
…
#elif defined(LINUX)
…
#endif
```
for the whole function, instead of the current three levels of imbrication.